### ISSUE TITLE
summary command

### ISSUE THREAD

**Issue by comqdhb** [tj/git-extras#764]
# error was 
```
 repo age : 10 years
 active   : 1082 days
 commits  : 4618
 files    : 3243
 authors  : 
/usr/bin/git-summary: line 82: column: command not found
```

# we needed to install the following to get git summary to work on debian:
```
apt-get install bsdmainutils
```

**Comment by spacewander** [comment]
@comqdhb 
I found out that `column` is not provided by default in some platforms.

The `column` is critical for aligning, because awk's printf doesn't handle the visual width of Unicode character correctly. Therefore I decide to update the documentation to add `column` as a dependency in https://github.com/tj/git-extras/pull/765

**Comment by spacewander** [comment]
Closed as #764 is merged.
